### PR TITLE
ui of authentication page

### PR DESCRIPTION
**Pull request by engeneee**
https://www.figma.com/file/ECUCBvpVvMgBLBdYWp9YRA/smavy?type=design&node-id=2-330&mode=design&t=ELkr9dmdxGm5rARq-0
photo_5438191344729314827_y
photo_5438191344729314825_y

### CODE REVIEW

File: lib/features/authentication/authentication.dart [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+  void dispose() {
+    _tabController.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return DefaultTabController(
+      length: 2,
+      child: Scaffold(
```
Comment (rzhaxibayev) [inline lib/features/authentication/authentication.dart]: Scaffold should be at the very top

File: lib/features/authentication/authentication.dart [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+                  style: TextStyle(
+                    color: Colors.grey,
+                    fontSize: 16.0,
+                  ),
+                ),
+              ),
+            ),
+          ],
+        ),
+        body: Padding(
```
Comment (rzhaxibayev) [inline lib/features/authentication/authentication.dart]: Wrap with scrollview

File: lib/features/authentication/authentication.dart [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+          padding: const EdgeInsets.all(16.0),
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.start,
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: <Widget>[
+              SizedBox(height: 16.0),
```
Comment (rzhaxibayev) [inline lib/features/authentication/authentication.dart]: const

File: lib/features/authentication/authentication.dart [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+              Text(
```
Comment (rzhaxibayev) [inline lib/features/authentication/authentication.dart]: fix all const warnings

File: lib/features/authentication/authentication.dart [hunk 5]
```diff
@@ -0,0 +1,282 @@
...
+                              // пароль востановить ету
+                            },
+                            child: Text(
+                              'Забыли пароль?',
+                              style: TextStyle(color: Colors.grey),
+                            ),
+                          ),
+                        ),
+                        SizedBox(height: 16.0),
+                        Container(
```
Comment (rzhaxibayev) [inline lib/features/authentication/authentication.dart]: Use SizedBox instead of Container

File: lib/features/authentication/authentication.dart [hunk 6]
```diff
@@ -0,0 +1,282 @@
...
+                              _tabController.animateTo(0); 
+                            },
+                            child: Text(
+                              'Уже есть аккаунт?',
+                              style: TextStyle(color: Colors.grey),
+                            ),
+                          ),
+                        ),
+                        SizedBox(height: 16.0),
+                        Container(
```
Comment (rzhaxibayev) [inline lib/features/authentication/authentication.dart]: SizedBox

File: pubspec.yaml [hunk 7]
```diff
@@ -35,6 +35,7 @@ dependencies:
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.2
+  material: 1.0.0+2
```
Comment (rzhaxibayev) [inline pubspec.yaml:38]: why needed?

**Review by rzhaxibayev — COMMENTED** [review]
Fix comments

File: pubspec.yaml (hunk 7 above)
Comment (engeneee) [reply pubspec.yaml:38]: without this dependency the code in main.dart did not work

**Pull request closed without merging**